Makefile

```diff
@@ -5,3 +5,12 @@ clean_docs:
 
 docs:
 	$(MAKE) -C docs html SPHINXOPTS="-W -n"
+
+clean:
+	find . -type d -name __pycache__ -print -exec rm -r {} \+
+
+deploy_firmware: clean
+	# requires rshell  https://github.com/dhylands/rshell
+	 rshell rsync software/firmware /pyboard/lib
```

Comment:
It looks like rsync doesn't support pattern matching, so this is going to copy the .MD files from contrib too. If we start to get close to the 2MB flash memory limit, we might want to use `cp` which does support matching, or look into a minification step.

https://github.com/dhylands/rshell#rsync
https://github.com/dhylands/rshell#cp

Reply:
I'm gonna do this now. It kinda makes us look sill to be copying files indiscriminately to the pico.